tests/model.test.js

```diff
@@ -1,12 +1,99 @@
 import { expect } from "chai";
 
-import { Model } from "../src/model";
+import { Model } from "../src/compound_model";
```

Comment:
Should this file be called `compound_model.test.js` then?

Reply:
Agreed, but I also think once we phased out the "old" model, we should refer to the "compound model" just as "model".